common/src/main/java/com/microsoft/identity/common/internal/platform/AndroidPlatformUtil.java

```diff
@@ -215,7 +215,7 @@ private boolean isValidHubRedirectURIForNAATests(String redirectUri) {
         // Below redirectURI is being used in our automation tests and also by OneAuth tests for NAA
         return BuildConfig.DEBUG && (redirectUri.equals("msauth://com.microsoft.teams/VCpKgbYCXucoq1mZ4BZPsh5taNE=")
                         || redirectUri.equals("msauth://com.microsoft.teams/fcg80qvoM1YMKJZibjBwQcDfOno=")
-                        || redirectUri.equals("msauth://com.microsoft.teams/fcg80qvoM1YMKJZibjBwQcDfOno%3D"));
+                        || redirectUri.equals("https://login.microsoftonline.com/common/oauth2/nativeclient"));
```

Comment:
Generally apps use the same redirect URI across clouds. Is there a reason Teams has adopted a different one for USGov? Also strange that this has `.com` in it and not `.us`.

Also good but interesting that this is an app link.

Reply:
You are right, they should do have same redirectURI for USGov and WW clouds. But, we could not use the common ones in our testing with our lab accounts. It says the redirectURI is not recognized in msaidlab4 tenant. So, I believe Nick has resolved this issue by getting the other redirectURI I added above. I am not aware of what was done to make this usable. But, this is mostly used by us and OneAuth team for testing only NAA scenarios.  So, I did not check more on this.